### PR TITLE
Add registry org env

### PR DESCRIPTION
**Pull request by mbthiery**
Quality of life change to help people outside of Helium create builds more easily.

### CODE REVIEW

**Comment by abhay** [comment]
Seems reasonable, @jadeallenx ?

**Comment by mbthiery** [comment]
@jadeallenx anything needed from me to move this forward?

File: .buildkite/scripts/make_image.sh [hunk 1]
```diff
@@ -25,7 +26,7 @@ if [[ ! $TEST_BUILD -eq "0" ]]; then
     REGISTRY_NAME="test-builds"
     DOCKER_BUILD_ARGS="--build-arg REBAR_DIAGNOSTIC=1 $DOCKER_BUILD_ARGS"
 fi
-MINER_REGISTRY_NAME="$REGISTRY_HOST/team-helium/$REGISTRY_NAME"
+MINER_REGISTRY_NAME="$REGISTRY_HOST/$REGISTRY_ORG/$REGISTRY_NAME"
```
Comment (madninja) [inline .buildkite/scripts/make_image.sh:29]: You'll want to set it for TEST_BUILD too. Otherwise looks good

**Review by jadeallenx — APPROVED** [review]
Seems ok

**Pull request merged**